Initium-Odp/src/com/universeprojects/miniup/server/controllers/GroupController.java

```diff
@@ -226,16 +226,18 @@ private int sortNumber(CachedEntity e) {
 			int activeUsersPastWeek = db.getActiveGroupPlayers(group, members, 60*24*7).size();
 			request.setAttribute("activeUsersPastWeek", activeUsersPastWeek);
 			
-			List<Key> keyOfDecs = GroupService.getWarDeclarations(group);
+			List<Key> keyOfDecs = (List<Key>)group.getProperty("declaredWarGroups");
 			List<String> warDecGroupNames = new ArrayList<String>();
-			if (!keyOfDecs.isEmpty())
+			if (keyOfDecs.isEmpty() || keyOfDecs == null)

```

Comment:
I meant you'll want to put the null check first. If it is null, this will throw an npe.